.eslintrc.yml

```diff
@@ -23,8 +23,8 @@ settings:
             - static
             - node_modules
           alias:
-            '#js': js
-            '#img': images
+            '@/js': js
+            '@/img': images
```

Comment:
It turns out the `#` prefix isn't a great choice, since it has special meaning as a hash for CSS urls.

Reply:
I wonder if we could get away with magic prefixes in general? It looks like webpack supports giving precedence to our own `src` folder over `node_modules` (see second example: https://webpack.js.org/configuration/resolve/#resolvemodules)

Reply:
Huh, looks like our webpack config is already configured that way. I assume just `js/...` doesn't work?

Reply:
Yeah, that's true, it would work that way. I have a slight preference for something noticeable that makes it clear when a module is being imported locally vs from an external package. We're also sorting imports, and this allows local modules to be grouped separately: https://github.com/lydell/eslint-plugin-simple-import-sort#custom-grouping

Reply:
That makes sense 👍